### PR TITLE
[HYD-202] Add instruction for `parquet_s3_fdw`

### PR DESCRIPTION
**Pull request by owenthereal**
Add instruction for `parquet_s3_fdw`

### CODE REVIEW

File: features/external_tables.md [hunk 1]
```diff
@@ -190,6 +190,62 @@ create foreign table users_csv (
...
+```
+
+Upload the parquet files to a S3 bucket folder called `sample-data`, and create a S3 Parquet foreign table, replacing `...` with your AWS credentials, region, and S3 bucket name:
+
+```sql
+CREATE EXTENSION parquet_s3_fdw;
+
+CREATE SERVER parquet_s3_srv FOREIGN DATA WRAPPER parquet_s3_fdw OPTIONS (aws_region '...');
+
+CREATE USER MAPPING FOR public SERVER parquet_s3_srv OPTIONS (user '...', password '...');
```
Comment (wuputah) [inline features/external_tables.md]:
```suggestion
CREATE USER MAPPING FOR CURRENT_USER SERVER parquet_s3_srv OPTIONS (user '...', password '...');
```

File: features/external_tables.md [hunk 2]
```diff
@@ -190,6 +190,62 @@ create foreign table users_csv (
...
+    salary FLOAT8,
+    title text,
+    comments text
+)
+SERVER parquet_s3_srv
+OPTIONS (
+    dirname 's3://.../sample-data'
+);
+```
+
```
Comment (wuputah) [inline features/external_tables.md:249]:
```suggestion
You can now read data from the Parquet file using `SELECT ... FROM userdata`. Note that every query will read the data again, incurring charges on your AWS account. For better performance and avoiding ongoing charges, we recommend caching the data locally in Hydra by insert it into a table or using a materialized view.

```

Please correct any factual inaccuracies I may have added here. 😁 

Are any other performance metrics or considerations we can mention?

**Review by wuputah — APPROVED** [review]

File: features/external_tables.md (hunk 1 above)
Comment (wuputah) [reply features/external_tables.md]: If there's any other use of `PUBLIC` in the docs, I'd recommend updating to `CURRENT_USER`.

File: features/external_tables.md (hunk 2 above)
Comment (owenthereal) [reply features/external_tables.md:249]: You are correct that every `SELECT` is going to fetch from s3 again. And if people don't want to fetch every time, they could use a materialized view.

**Pull request merged**